target/linux/stm32/patches-6.6/900-ARM-dts-stm32-add-ethernet2-for-STM32MP135F-DK-board.patch

```diff
@@ -0,0 +1,51 @@
+From 63c84f846ec5b794a6c90a1c5813cdb0ae75daf6 Mon Sep 17 00:00:00 2001
+From: Thomas Richard <thomas.richard@bootlin.com>
+Date: Thu, 26 Sep 2024 16:48:52 +0200
+Subject: [PATCH] ARM: dts: stm32: add ethernet2 for STM32MP135F-DK board
+
+Signed-off-by: Thomas Richard <thomas.richard@bootlin.com>
```

Comment:
Are you planning to add this to upstream Linux kernel too?

Reply:
Which of these patches are not yet in linus mainline tree?

Reply:
- 020-mmc-mmci-stm32-add-SDIO-in-band-interrupt-mode.patch

>  Is available upstream, but I cherry-picked the commit from the ST tree. For next iteration, I updated the patch to use the upstream commit.

- 100-ARM-dts-stm32-rtc-add-pin-to-provide-LSCO-on-stm32mp.patch
- 101-ARM-dts-stm32-rtc-add-LSCO-to-WLAN-BT-module-on-stm3.patch
- 102-ARM-dts-stm32-add-support-of-WLAN-BT-on-stm32mp135f-.patch

> These three patches were sent on mailing list, but not accepted, when I created the pull request. Now they are in the pull request sent by Alexandre Torgue for 6.13 (https://lore.kernel.org/all/92d2d6df-cc5c-488f-8ebd-550b1903db12@foss.st.com/). We can assume that they will be merged upstream during next merge window. For next iteration I updated patches to use commits from the pull request and I renamed them as 03x- (we can consider they are upstream backports).

- 700-net-stmmac-dwmac-stm32-add-support-of-phy-supply-pro.patch

> This patch is needed to enable the support of ethernet2. There is nothing available upstream.
I wrote it using ST linux 6.1 as reference.
Now ST has released 6.6 kernel, so there is a patch that I can easily backport.
For next iteration, I updated the patch to use this new one.

- 900-ARM-dts-stm32-add-ethernet2-for-STM32MP135F-DK-board.patch

>Same than for 700-net-stmmac-dwmac-stm32-add-support-of-phy-supply-pro.patch.

For these 2 patches (700 and 900), there is nothing available upstream. But ST planned to do the job (as mentioned in the commit message https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/arch/arm/boot/dts/st/stm32mp135f-dk.dts?id=e9442f1fa4d2545dd6c0aaf7cb7a125cb04f8f2f). So I'm not planning to upstream these two patches.

Other patches are already available upstream.

Reply:
Thanks for the update. This looks fine. If they just backports we do not have to maintain the patches for ever.